### PR TITLE
Adds holy vision (Some mobs can now see blessed tiles)

### PR DESCRIPTION
**Pull request by DaxDupont**
[Changelogs]: 
:cl: Dax Dupont & Coiax
add: Praise the lord for he has granted thy chaplains, wizards, revenants and eminences with holy vision. They can now see blessed tiles.
/:cl:

[why]: Finally we can see blessed tiles again. I think it was a feature at one point that got removed, not sure what happened to it but the wiki still mentions it.
I opted out observers aside from the ones that inherit the view from their former being to not have holiness shown all over the place.
dreamseeker_2018-01-16_00-37-48

### CODE REVIEW

**Comment by MoreRobustThanYou** [comment]
Idea: have the holy turfs use connected textures so they'd look something like
untitled

**Comment by DaxDupont** [comment]
I don't think you can even do that in this case

**Comment by MoreRobustThanYou** [comment]
anything's possible

**Comment by coiax** [comment]
Can you have it as a toggle? And let observers have that toggle?

**Comment by DaxDupont** [comment]
The way alternate appearances works makes it hard to make it a toggle, since you can't just toggle it back on without looping through all blessing effects and refreshing them.
I really don't think that's a good idea.

**Comment by deathride58** [comment]
Could probably use a planemaster to make turning it into a toggle a lot easier, along with making it a lot lighter on server performance.

**Comment by DaxDupont** [comment]
I don't know how to use a planemaster

**Comment by Armhulen** [comment]
that's pretty gross i'll be honest

**Comment by Armhulen** [comment]
just do some shimmering yellowness it has to be better than that

**Comment by Bawhoppen** [comment]
Wizards and chaplains should not get this...
That's a huge balance change. 
The sprite is also very bad and needs to be way better before this ever got approved. Also what MRTY is suggesting would be far preferable and is simply just using the smoothing flags.

**Comment by DaxDupont** [comment]
I would like someone to give me a good sprite
Also I don't think you can smooth effects, and I dont think it'll work with alternate appearances

**Comment by DaxDupont** [comment]
I can just reuse the clockwork floor glow

**Comment by DaxDupont** [comment]
Here I made it so it's a modified ratvar floor glow that just glimmers occasionally.

**Comment by DaxDupont** [comment]
End result:
Every 10 seconds:
https://i.imgur.com/2anvSPT.gif

**Comment by coiax** [comment]
@DaxDupont actually, I think all observers use the alternate appearance system for toggling on and off people's individual ghost cosmetics (which is up to the viewer whether they see it), so I don't think it would be that bad to have a toggle.

**Comment by coiax** [comment]
Also, I like the new sprite. Hurray for reusing animations, it's what I did for shuttle ripples.

File: code/game/alternate_appearance.dm
Comment (coiax) [inline code/game/alternate_appearance.dm]: also include cult wraiths

**Comment by Potato-Masher** [comment]
I believe the wiki says it was a faint gold shimmer every once in a while. I agree that we might need a little less obnoxious effect/sprite. 

The only thing I'm a little unsure about is the wizard getting this vision. Maybe add the holy vision when the wizard gets the Scrying orb as an add on to his xray vision?

**Comment by DaxDupont** [comment]
This is the new sprite @Potato-Masher, people were complaining about the old one

**Comment by ChangelingRain** [comment]
I'd recommend against using this visual for these. It's especially odd for the clockcult camera mob thing, since that visual means *good things* to it.

**Comment by DaxDupont** [comment]
Honestly unless someone comes up with something else I can use I'd keep it like this tbh. 
I can't sprite well enough.

**Comment by Bawhoppen** [comment]
Why are you trying to make a change if you can't actually get the sprite for it then?

**Comment by DaxDupont** [comment]
@Bawhoppen Because @coiax his one went stale. We were supposed to have this feature years ago, it's been on the wiki for ages. Besides I tried using my own sprite but people didn't like it, which was about the best I could do.
It would just be nice if the people whining about the sprite being bad do more than just whine about it. 
I want the gameplay mechanic and the way it looked originally was good for me, this is also fine for me. It's a matter of taste.

**Comment by DaxDupont** [comment]
There now it's barely visible sparkling on tiles.

**Comment by Potato-Masher** [comment]
Ooo, sparkly. I like that one, it isn't too hard on the eyes and shouldn't be too annoying when you bless the entire main hallway or something.

I ain't no spriter, so you should take my opinions with a grain of salt though. I'm surprised that none of the usual spriters are giving feedback though, they are way better at telling whats good or bad.

**Comment by Bawhoppen** [comment]
Having sprite standards is not autism

**Comment by DaxDupont** [comment]
Can you review @ShizCalev it's been open for 4 days

File: code/game/objects/effects/blessing.dm
Comment (Cyberboss) [inline code/game/objects/effects/blessing.dm]: else is dangling

File: code/modules/mob/mob_movement.dm
Comment (Cyberboss) [inline code/modules/mob/mob_movement.dm]: if locate()...

File: code/modules/spells/spell_types/ethereal_jaunt.dm
Comment (Cyberboss) [inline code/modules/spells/spell_types/ethereal_jaunt.dm]: if locate() ...

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by DaxDupont** [comment]
Done @Cyberboss

**Pull request merged**
